ArcGISRuntimeSDKQt_CppSamples/EditData/DeleteFeaturesFeatureService/DeleteFeaturesFeatureService.h

```diff
@@ -49,13 +46,11 @@ class DeleteFeaturesFeatureService : public QQuickItem
   Q_INVOKABLE void deleteSelectedFeature();
 
 signals:
-  void calloutDataChanged();
```

Comment:
This still works correctly with this?

Reply:
Yes. Tested all of them. The CalloutData pointer itself doesn't change but the underlying properties in the CalloutData do, and those all emit